src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,481 @@
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUSTOMER_APPLICATIONS" AS SELECT * FROM "SDT_OWNER"."BULK_CUSTOMER_APPLICATIONS";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUST_APP_SEQ" AS SELECT * FROM "SDT_OWNER"."BULK_CUST_APP_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_CUST_SEQ" AS SELECT * FROM "SDT_OWNER"."BULK_CUST_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_SUBMISSIONS" AS SELECT * FROM "SDT_OWNER"."BULK_SUBMISSIONS";
+CREATE OR REPLACE VIEW "SDT_USER"."BULK_SUB_SEQ" AS SELECT * FROM "SDT_OWNER"."BULK_SUB_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."ERROR_LOGS" AS SELECT * FROM "SDT_OWNER"."ERROR_LOGS";
+CREATE OR REPLACE VIEW "SDT_USER"."ERROR_MESSAGES" AS SELECT * FROM "SDT_OWNER"."ERROR_MESSAGES";
+CREATE OR REPLACE VIEW "SDT_USER"."ERR_LOG_SEQ" AS SELECT * FROM "SDT_OWNER"."ERR_LOG_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."ERR_MESG_SEQ" AS SELECT * FROM "SDT_OWNER"."ERR_MESG_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."GLB_PAR_SEQ" AS SELECT * FROM "SDT_OWNER"."GLB_PAR_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."GLOBAL_PARAMETERS" AS SELECT * FROM "SDT_OWNER"."GLOBAL_PARAMETERS";
+CREATE OR REPLACE VIEW "SDT_USER"."INDIVIDUAL_REQUESTS" AS SELECT * FROM "SDT_OWNER"."INDIVIDUAL_REQUESTS";
+CREATE OR REPLACE VIEW "SDT_USER"."IND_REQ_SEQ" AS SELECT * FROM "SDT_OWNER"."IND_REQ_SEQ";
+--CREATE OR REPLACE VIEW "SDT_USER"."MESSAGE_LOGS" AS SELECT * FROM "SDT_OWNER"."MESSAGE_LOGS";
+CREATE OR REPLACE VIEW "SDT_USER"."SDT_REF_SEQ" AS SELECT * FROM "SDT_OWNER"."SDT_REF_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."SERVICE_REQUESTS" AS SELECT * FROM "SDT_OWNER"."SERVICE_REQUESTS";
+CREATE OR REPLACE VIEW "SDT_USER"."SERVICE_ROUTINGS" AS SELECT * FROM "SDT_OWNER"."SERVICE_ROUTINGS";
+CREATE OR REPLACE VIEW "SDT_USER"."SERVICE_TYPES" AS SELECT * FROM "SDT_OWNER"."SERVICE_TYPES";
+CREATE OR REPLACE VIEW "SDT_USER"."SER_ROU_SEQ" AS SELECT * FROM "SDT_OWNER"."SER_ROU_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."SER_TYP_SEQ" AS SELECT * FROM "SDT_OWNER"."SER_TYP_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."SRV_REQ_SEQ" AS SELECT * FROM "SDT_OWNER"."SRV_REQ_SEQ";
+CREATE OR REPLACE VIEW "SDT_USER"."TARGET_APPLICATIONS" AS SELECT * FROM "SDT_OWNER"."TARGET_APPLICATIONS";
+CREATE OR REPLACE VIEW "SDT_USER"."TAR_APP_SEQ" AS SELECT * FROM "SDT_OWNER"."TAR_APP_SEQ";
+
+------------------------------------------------
+-- Create Views (Not Synonyms!) for SDT_BATCH_USER
+------------------------------------------------
+CREATE OR REPLACE VIEW "SDT_BATCH_USER"."BULK_SUBMISSIONS" AS SELECT * FROM "SDT_OWNER"."BULK_SUBMISSIONS";
+CREATE OR REPLACE VIEW "SDT_BATCH_USER"."ERROR_LOGS" AS SELECT * FROM "SDT_OWNER"."ERROR_LOGS";
+CREATE OR REPLACE VIEW "SDT_BATCH_USER"."GLOBAL_PARAMETERS" AS SELECT * FROM "SDT_OWNER"."GLOBAL_PARAMETERS";
+CREATE OR REPLACE VIEW "SDT_BATCH_USER"."INDIVIDUAL_REQUESTS" AS SELECT * FROM "SDT_OWNER"."INDIVIDUAL_REQUESTS";
+--CREATE OR REPLACE VIEW "SDT_BATCH_USER"."PURGE" AS SELECT * FROM "SDT_OWNER"."PURGE";
```

Comment:
Remember to remove this commented out line.

Reply:
May be reintroduced as part of SDT-19 so keeping, for now,